src/uct/api/uct.h

```diff
@@ -1067,6 +1068,39 @@ ucs_status_t uct_md_mem_alloc(uct_md_h md, size_t *length_p, void **address_p,
  */
 ucs_status_t uct_md_mem_free(uct_md_h md, uct_mem_h memh);
 
+/**
+ * @ingroup UCT_MD
+ * @brief list of UCT memory use advice
+ *
+ */
+typedef enum uct_mem_advice {
+    UCT_MADV_NORMAL  = 0,  /**< No special treatment */
+    UCT_MADV_WILLNEED      /**< can be used on the memory mapped with the
```

Comment:
s/ the$//

minor cleanup for readability so that it reads:
  can be used on the memory mapped with